src/ui/atoms/text/Text.tsx

```diff
@@ -12,42 +12,49 @@ export type TextProps = DeepReadonly<{
   /**
    * The size of the rendered text.
    */
-  readonly size?: 'x-large' | 'large' | 'medium' | 'small' | 'x-small'
+  readonly fontSize?: 'x-large' | 'large' | 'medium' | 'small' | 'x-small'
   /**
    * `brand` (Gotham) is the main font for ØKP4 branding,
    * therefore it is mainly used for all UI interfaces.
    * The `secondary` font is Noto Sans Mono.
    */
   readonly fontFamily?: 'brand' | 'secondary'
   /**
-   * The different font weights declared for each font-face.
+   * The different font weights declared for each font-family.
    */
   readonly fontWeight?: 'light' | 'normal' | 'bold' | 'black'
   /**
    * The color applied to the final rendering of the typography,
    * regarding the current theme and following semantic color naming.
    */
   readonly color?: 'text' | 'highlighted-text' | 'success' | 'warning' | 'error' | 'info'
+  /**
+   * Handles the `white-space` property in the element.
+   * If true, the text will not wrap, but instead will truncate with a text overflow ellipsis.
+   */
+  readonly noWrap?: boolean
```

Comment:
@ccamel I will say that the most used property remains `ellipsis` because `clip` sharply cuts the text flush with its container (very rare utility) and `fade` remains an experimental property which is very poorly supported by browsers.
![Capture d’écran 2022-04-12 à 14 42 12](https://user-images.githubusercontent.com/47554752/162964696-336e9fba-7f89-4d45-a430-137757e61153.png)

Reply:
All right. Good for me! Thx. 😊